plugins/git/service/src/gitservice.cpp

```diff
@@ -94,42 +92,39 @@ void GitServiceHandler::getRepositoryList(std::vector<GitRepository>& return_)
        dirIter != endIter;
        ++dirIter)
   {
-    if (fs::is_directory(dirIter->status()))
+    // TODO: What else can this be if not a directory?
```

Comment:
We discussed this to, we will store a config file here which will contain the repository name for the unique ids. So remove this TODO.